cmd/limactl/disk.go

```diff
@@ -0,0 +1,305 @@
+`,
+		Short: "Create a Lima disk",
+		Args:  cobra.ExactArgs(1),
+		RunE:  diskCreateAction,
+	}
+	diskCreateCommand.Flags().String("size", "", "configure the disk size")
+	diskCreateCommand.MarkFlagRequired("size")
+	return diskCreateCommand
+}
+
+func diskCreateAction(cmd *cobra.Command, args []string) error {
+	size, err := cmd.Flags().GetString("size")
+	if err != nil {
+		return err
+	}
+
+	diskSize, err := units.RAMInBytes(size)
+	if err != nil {
+		return err
+	}
+
+	// only exactly one arg is allowed
+	name := args[0]
+
+	diskDir, err := store.DiskDir(name)
+	if err != nil {
+		return err
+	}
+
+	if _, err := os.Stat(diskDir); !errors.Is(err, fs.ErrNotExist) {
+		return fmt.Errorf("disk %q already exists (%q)", name, diskDir)
+	}
+
+	logrus.Infof("Creating a disk %q", name)
+
+	if err := os.MkdirAll(diskDir, 0700); err != nil {
+		return err
+	}
+
+	if err := qemu.CreateDataDisk(diskDir, int(diskSize)); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func newDiskListCommand() *cobra.Command {
+	var diskListCommand = &cobra.Command{
+		Use: "list",
+		Example: `
+To list existing disks:
+$ limactl disk list
+`,
+		Short:   "List existing Lima disks",
+		Aliases: []string{"ls"},
+		Args:    cobra.NoArgs,
+		RunE:    diskListAction,
+	}
+	diskListCommand.Flags().Bool("json", false, "JSONify output")
+	return diskListCommand
+}
+
+func diskListAction(cmd *cobra.Command, args []string) error {
+	jsonFormat, err := cmd.Flags().GetBool("json")
+	if err != nil {
+		return err
+	}
+
+	allDisks, err := store.Disks()
+	if err != nil {
+		return err
+	}
+
+	if jsonFormat {
+		for _, diskName := range allDisks {
+			disk, err := store.InspectDisk(diskName)
+			if err != nil {
+				logrus.WithError(err).Errorf("disk %q does not exist?", diskName)
+				continue
+			}
+			j, err := json.Marshal(disk)
+			if err != nil {
+				return err
+			}
+			fmt.Fprintln(cmd.OutOrStdout(), string(j))
+		}
+		return nil
+	}
+
+	w := tabwriter.NewWriter(cmd.OutOrStdout(), 4, 8, 4, ' ', 0)
+	fmt.Fprintln(w, "NAME\tSIZE\tDIR\tIN USE BY")
+
+	if len(allDisks) == 0 {
+		logrus.Warn("No disk found. Run `limactl disk create DISK --size SIZE` to create a disk.")
+	}
+
+	for _, diskName := range allDisks {
+		disk, err := store.InspectDisk(diskName)
+		if err != nil {
+			logrus.WithError(err).Errorf("disk %q does not exist?", diskName)
+			continue
+		}
+		fmt.Fprintf(w, "%s\t%s\t%s\t%s\n", disk.Name, units.BytesSize(float64(disk.Size)), disk.Dir, disk.Instance)
+	}
+
+	return w.Flush()
+}
+
+func newDiskDeleteCommand() *cobra.Command {
+	var diskDeleteCommand = &cobra.Command{
+		Use: "delete DISK [DISK, ...]",
+		Example: `
+To delete a disk:
+$ limactl disk delete DISK
+
+To delete multiple disks:
+$ limactl disk delete DISK1 DISK2 ...
+`,
+		Aliases: []string{"remove", "rm"},
+		Short:   "Delete one or more Lima disks",
+		Args:    cobra.MinimumNArgs(1),
+		RunE:    diskDeleteAction,
+	}
+	diskDeleteCommand.Flags().Bool("force", false, "force delete")
```

Comment:
The option should allow `-f` shorthand, just like `limactl delete -f INSTANCE`:

```suggestion
	diskDeleteCommand.Flags().BoolP("force", "f", false, "force delete")
```